agent/installer/installer.go

```diff
@@ -52,11 +52,18 @@ func getSupportedRegistry(ob algo.OutputBuilder) registry {
 
 		// BYOH Bundle Repository. Associate bundle with installer
 		linuxDistro := "Ubuntu_20.04.1_x86-64"
-		addBundleInstaller(linuxDistro, "v1.22.3", &algo.Ubuntu20_4K8s1_22{})
+		// If no version specified use the minimum v1.22 supported
+		if k8sVer == "" {
+			k8sVer = "v1.22"
+		}
+		addBundleInstaller(linuxDistro, k8sVer, &algo.Ubuntu20_4K8s1_22{})
```

Comment:
Here, we can hard code to a particular minor series - 
`addBundleInstaller(linuxDistro, "v1.22.*", &algo.Ubuntu20_4K8s1_22{})`